sherpa-onnx/python/tests/testdata/tokens_cn.txt

```diff
@@ -0,0 +1,5539 @@
+<blk> 0
```

Comment:
Sure, I put them in the repo, because I think they are small.  I think github also supports lfs, can we put it there?

Reply:
I don't think you need to use `git lfs` to manage them. You can upload them to github directly without `git lfs`.

Reply:
> Sure, I put them in the repo, because I think

If you want to use GitHub, you can create a repo for it. 
The point is that we can download it from GitHub actions during the test.